gustaf/helpers/raise_if.py

```diff
@@ -47,16 +47,15 @@ class ModuleImportRaiser():
     them to function. Examples are `splinepy` and `vedo`.
     """
 
-    def __init__(self, lib_name: str, error_message: Optional[str] = None):
+    def __init__(self, lib_name: str, error_message: Optional[str] = ""):
         self._message = str(
-        "Parts of the requested functionality in gustaf depend on the "
-        f"external `{lib_name}` package which could not be found on "
-        "your system. Please refer to the installation instructions "
-        "for more information. "
-        f"{f'Original error message {error_message}' if error_message else ''}"
+                "Parts of the requested functionality in gustaf depend on the "
+                f"external `{lib_name}` package which could not be found on "
+                "your system. Please refer to the installation instructions "
+                "for more information. "
+                f"Original error message - {error_message}"
```

Comment:
This will now always print the Original error message part even if none is given.

Reply:
what about now?

Reply:
we can probably replace documentation address with `pip` or `conda` commands